answer-01

```diff
@@ -0,0 +1,49 @@
+###### 1. What is the difference between private limited and public limited company? State one example for each type.
+
+A private limited company is a closely held one and requires at least two or more persons, for its formation. 
+For example, Petronas Sdn. Bhd. 
+
+A public limited company is owned and traded publicly. It requires seven persons for its setup.
+For example, Maybank Bhd.
+
+ `https://keydifferences.com/difference-between-public-company-and-private-company.html`
+
+###### 2. Why is Berjaya Sports Toto Berhad registered under Non-Shariah Compliant?
+
+I think Toto is one of the example of gambling/betting and this is against the teaching of muslim, hence it is registered under Non-Shariah Compliant.
+
+###### 3.List down 3 Non-Shariah Compliant Business Activities.
+- Alcohol
+- Gambling
+- Prostitution
+
+`https://av.sc.com/my/content/docs/Non-Syariah-Compliant-Business-Activities_ref.pdf` 
+
+###### 4.Name the stock exchange of Malaysia.
+
+Bursa Malaysia
+
+###### 5.Create a new folder name 'AR' in your repo main branch. Upload Annual Report 2018 - 2020 for companies below:
+(a) ViTrox Corp
+(b) Pentamaster Corp
+*There should be at least 6 PDF files.
+
+###### 6.Who are the users of annual reports?
+Current shareholders and potential investors
+
+`https://www.inc.com/encyclopedia/annual-reports.html#:~:text=Current%20shareholders%20and%20potential%20investors,large%20are%20also%20targeted%20audiences.`
+
+###### 7.What is the most important part of an annual report?
+
+I think the most important part is the financial statement to allow current and future investors to determine how the company has performed in past.
+
+###### 8.You have RM 10,000 savings and want to purchase shares as an investment for earning side income. You have listed two potential companies and need to select one company to invest. Comparing Pentamaster Corporation Bhd. and ViTrox Corporation Bhd. Justify your investment choice.
+
+Pentamaster. According to the link below, Pentamaster is a better investment choice in terms of its liquidity, growth and valuation.
```

Comment:
Good attempt. You will learn more aspects of financial analysis other than the ratios that you mentioned above from AKW104. This will help you justify the decision better.